internal/expr/bindinputs.go

```diff
@@ -50,10 +50,13 @@ func (tbe *TypeBoundExpr) BindInputs(args ...any) (pq *PrimedQuery, err error) {
 				return nil, err
 			}
 			argTypeUsed[te.input.ArgType()] = true
-			for _, val := range vals {
+			for i, val := range vals {
 				namedInput := sql.Named("sqlair_"+strconv.Itoa(inputCount), val.Interface())
 				params = append(params, namedInput)
 				sqlStr.WriteString("@sqlair_" + strconv.Itoa(inputCount))
+				if i < len(vals)-1 {
```

Comment:
You could also write this at the top of the loop if `i != 0`, which would save the `len` call each time.